docs/cloud/how-to-get-started-with-temporal_cloud.md

```diff
@@ -0,0 +1,99 @@
+---
+id: how-to-get-started-with-temporal-cloud
+title: How to get started with Temporal Cloud
+sidebar_label: Get started
+description: Create an account, get certificates, create a Namespace, and invite users.
+tags:
+  - how-to
+---
+
+<!--- Onboarding guide for Temporal Cloud --->
+
+To get started with Temporal Cloud, following these onboarding steps:
+
+1. [Create an account.](#create-an-account-in-temporal-cloud)
+1. [Get CA certificates.](#get-ca-certificates)
```

Comment:
Instead of `Get` CA Certificates (which seems to imply Temporal hands them out...we've had customers think that is something we do)...maybe we should `Secure CA Certificates from your organization` or something? We want to strongly state it in this inline text not just in the description IMO

Reply:
Agree with Mike. Keeping it simple, "Generate CA Certificates" would suffice, and we can link to our certs page for more info.